src/features/defi/components/LearnMore/LearnMore.tsx

```diff
@@ -0,0 +1,131 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, IconButton } from '@chakra-ui/react'
+import { useSteps } from 'chakra-ui-steps'
+import rewards from 'assets/rewards.svg'
+import risk from 'assets/risk.svg'
+import withdraw from 'assets/withdraw.svg'
+import { CarouselDots } from 'components/CarouselDots/CarouselDots'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+
+import { DefiModalHeader } from '../DefiModalHeader/DefiModalHeader'
+
+const STEPS_LENGTH = 3
+const STEP_TO_ELEMENTS_MAPPING = {
+  1: {
+    bodies: [
+      'defi.learnMore.bodies.rateFluctuationInfo',
+      'defi.learnMore.bodies.amountStakingInfo',
+      'defi.learnMore.bodies.withdrawInfo'
+    ],
+    headerImageSrc: rewards
+  },
+  2: {
+    bodies: ['defi.learnMore.bodies.unbondingInfo'],
+    headerImageSrc: withdraw
+  },
+  3: {
+    bodies: ['defi.learnMore.bodies.slashingInfo', 'defi.learnMore.bodies.partnerInfo'],
+    headerImageSrc: risk
+  }
+}
+
+type LearnMoreProps = {
+  assetId: string
+}
+
+export const LearnMore = ({ assetId }: LearnMoreProps) => {
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmo'
+  }))(assetId)
+
+  const handleClose = () => 'TODO'
+
+  const { nextStep, prevStep, activeStep } = useSteps({
+    initialStep: 1
+  })
+
+  const isLastStep = activeStep === STEPS_LENGTH
+  const isFirstStep = activeStep === 1
+
+  const handleNextOrCloseClick = () => {
+    if (isLastStep) return handleClose()
+
+    nextStep()
+  }
+
+  const handlePrevClick = () => {
+    if (isFirstStep) {
+      // TODO
```

Comment:
Since this is the barebones component without the routing specifics included, there's nothing done here for now.